scripts/qesap/lib/config.py

```diff
@@ -20,58 +40,76 @@ def yaml_to_tfvars(yaml_data):
     """
     config_out = ''
     terraform_variables = yaml_data['terraform']['variables']
-    print(yaml_data)
+    log.debug(yaml_data)
     for key, value in terraform_variables.items():
-        if isinstance(value, (str, int)):
-            entry = f'{key} = "{str(value)}"'
-        elif isinstance(value, bool):
-            entry = f'{key} = "{str(value).lower()}"'
-        elif isinstance(value, list):
-            entry = '", "'.join(value)
-            entry = f'{key} = ["{entry}"]'
-        elif isinstance(value, dict):
-            param_value = ''
-            for dict_key, dict_value in value.items():
-                param_value = f'{param_value}\t{dict_key} = "{dict_value}"\n'
-            entry = f'{key} = {{\n' \
-                    f'{param_value}' \
-                    f'}}'
-        else:
+        res, entry = yaml_to_tfvars_entry(key, value)
+        if not res:
             log.error(f'Unrecognized value type in yaml file: {key} = {value}')
-            return False
-        config_out = f'{config_out}\n{entry}'
+            return False, ''
+        config_out += f'\n{entry}'
 
     return config_out
 
 
+def terraform_yml(configure_data):
+    """
+    Check if Terraform:variables are present in the config.yaml
+    """
+    if not configure_data:
+        log.error("No configure_data")
+        return False
+
+    if 'terraform' not in configure_data.keys():
```

Comment:
`.keys()` here is redundant since Python 3.

Reply:
I like this particular redundancy :-)